internal/pkg/inject/offset_results.json

```diff
@@ -3798,25 +3798,808 @@
+                      "1.16.15",
+                      "1.17.0",
+                      "1.17.1",
+                      "1.17.2",
+                      "1.17.3",
+                      "1.17.4",
+                      "1.17.5",
+                      "1.17.6",
+                      "1.17.7",
+                      "1.17.8",
+                      "1.17.9",
+                      "1.17.10",
+                      "1.17.11",
+                      "1.17.12",
+                      "1.17.13",
+                      "1.18.0",
+                      "1.18.1",
+                      "1.18.2",
+                      "1.18.3",
+                      "1.18.4",
+                      "1.18.5",
+                      "1.18.6",
+                      "1.18.7",
+                      "1.18.8",
+                      "1.18.9",
+                      "1.18.10",
+                      "1.19.0",
+                      "1.19.1",
+                      "1.19.2",
+                      "1.19.3",
+                      "1.19.4",
+                      "1.19.5",
+                      "1.19.6",
+                      "1.19.7",
+                      "1.19.8",
+                      "1.19.9",
+                      "1.19.10",
+                      "1.19.11",
+                      "1.19.12",
+                      "1.19.13",
+                      "1.20.0",
+                      "1.20.1",
+                      "1.20.2",
+                      "1.20.3",
+                      "1.20.4",
+                      "1.20.5",
+                      "1.20.6",
+                      "1.20.7",
+                      "1.20.8",
+                      "1.20.9",
+                      "1.20.10",
+                      "1.20.11",
+                      "1.20.12",
+                      "1.20.13",
+                      "1.20.14",
+                      "1.21.0",
+                      "1.21.1",
+                      "1.21.2",
+                      "1.21.3",
+                      "1.21.4",
+                      "1.21.5",
+                      "1.21.6",
+                      "1.21.7",
+                      "1.21.8",
+                      "1.21.9",
+                      "1.21.10",
+                      "1.21.11",
+                      "1.21.12",
+                      "1.21.13",
+                      "1.22.0",
+                      "1.22.1",
+                      "1.22.2",
+                      "1.22.3",
+                      "1.22.4",
+                      "1.22.5",
+                      "1.22.6",
+                      "1.22.7",
+                      "1.22.8",
+                      "1.22.9",
+                      "1.23.0",
+                      "1.23.1",
+                      "1.23.2",
+                      "1.23.3"
+                    ]
+                  }
+                ]
+              }
+            ]
           }
         ]
-      }
-    ]
-  },
-  {
-    "module": "std",
-    "packages": [
+      },
       {
-        "package": "bufio",
```

Comment:
I might be miss-reading the diff, but it seems like the `"bufio".Writer.buf` offset is being removed here. Is that intentional?

Reply:
That threw me off too, but I think it was just moved above (if you search for it it's still there, please double check me)

Reply:
Ah, yeah, you are correct. All good. :+1: